app/models/comment.rb

```diff
@@ -314,4 +315,8 @@ def notify_slack_channel_about_warned_users
   def parent_exists?
     parent_id && Comment.exists?(id: parent_id)
   end
+
+  def send_web_mentions
+    WebMentions::SendWebMention.perform_async(id)
```

Comment:
In DEV's database out of 196k published articles, only 47k have canonical URLs which makes me think enqueueing this job for every single comment might be a waste of resources. What do you think about checking for a commentable canonical URL before enqueueing this job to determine if we even need it?